prune unused assets

kusma opened the pull request:
These are no longer used, mostly because they've been replaced with newer, higher quality versions of the asset, either as a PNG or SVG.

kusma committed:
prune unused assets

These are no longer used, mostly because they've been replaced with
newer, higher quality versions of the asset, either as a PNG or SVG.

kusma commented:
Just to be clear, these are assets that were already unused *before* I started moving things around!

lra reviewed (commented):
Thanks. What was the process to check that those files were not used?

kusma commented:
Lots and lots of grepping :laughing:

kusma commented on styles/001/gfx/font.png:
Whoops, this file is still in use in `img_ascii.php`, need to drop...

kusma replied:
Same with the `fntX.png` files, it seems.

kusma replied:
Oh, but img_ascii.php isn't in use anymore either, *and* it has bitrotted and no longer works... Oh well, that can be cleaned out separately.

kusma replied:
I should have remembered that, because I worked on the new asciiviewer :facepalm:

kusma committed:
remove unused fonts

The only user of this is img_ascii.php, which is no longer in use, and
no longer works anyway. Let's just rip this out as well.

kusma committed:
drop maintenance.html

This doesn't seem like the best way to do a maintenence-message.

While we're at it, garbage collect the gfx-symlink, trumpet.gif and
lolsplash.png, all of which are now unused.

lra committed:
Merge branch 'master' into cleanup

lra reviewed (approved):
Checked the files removed:    
   - gfx - a symlink, not used at root level
   - lolsplash.png, maintenance.html - not referenced
   - styles/001/gfx/*.gif and *.png variants - replaced by .svg versions
   - styles/001/gfx/logos.removed/* - legacy logos already marked as removed
   - styles/001/gfx/os/k_*.gif - old platform icons not used (likely replaced by types.png sprite)
   - Other misc files (fnt*.png, font.png, cdcstack_*.gif, etc.) - no references found
